src/pages/CreateEventPage/CreateEventPage.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useState } from 'react';
+import { CreateEventWrapper, Form } from './styled';
+import { parseISO, formatISO, format } from 'date-fns';
+import createEvent from './components/createEvent';
+import isFormFilled from './components/isFormFilled';
+import { InputField, TextArea, Select } from './components/FormFields';
+
+function CreateEvent() {
+  const [name, setName] = useState('');
+  const [image, setImage] = useState('');
+  const [imageAlt, setImageAlt] = useState('');
+  const [time, setTime] = useState(formatISO(Date.now()));
+  const [description, setDescription] = useState('');
+  const [eventType, setEventType] = useState('Choose event type');
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const [email, setEmail] = useState('');
+  let createWasSuccess = false;
+  const eventOptions: string[] = [
+    "Subject matter event",
+    "Coding event",
+    "Project presentation",
+    "Workshop"
+  ]
+
+  function submitEvent() {
+    createEvent(time, name, description, firstName, lastName, email)
+      .then((result) => {
+        createWasSuccess = true;
+      })
+      .catch(() => {
+        console.log('Create failed');
+      });
+  }
+
+  return (
+    <CreateEventWrapper>
+      <h3>Create new event</h3>
+      <Form>
+        <InputField
+          id="name"
+          name="name"
+          label="Event name"
+          type="text"
+          value={name}
+          onChange={setName}
+        />
+
+        <InputField
+          id="date"
+          name="date"
+          label="Date"
+          type="date"
+          value={format(parseISO(time), 'yyyy-MM-dd')}
+          onChange={(e: HTMLInputElement) =>
+            setTime(
+              formatISO(Date.parse(e + ' ' + format(parseISO(time), 'HH:mm')))
+            )
+          }
+        />
```

Comment:
This implementation of `value` and `onChange` will probably not work for users who want to type in the whole date manually. I tested it and it seems to throw an error since the value won't always a valid date mid-typing.